RELEASE-NOTES.rst

```diff
@@ -18,6 +18,293 @@
+-  Adds a new form for ILL requests, available for the patron through
+   the public interface. This form allows the patron to place a request
+   for a document not available in the patron’s organisation, and to
+   select the pickup library.
+-  Improves the loan API performance to render the patron account
+   (public and Professional) quicker.
+-  Improves the performance of the patron account information in the
+   checkin-checkout: a first call is requested to get linked item pids
+   and barcode, then for each barcode, a second call is done to get the
+   item details.
+
+Metadata
+~~~~~~~~
+
+-  Improves the regular expression used in the JSON schema to validate
+   dates, in all relevant resources.
+-  Create the new “collection” resource, to group item together.
+-  Adds optional fields to holdings that are displayed on the document
+   detailed view:
+
+   -  ``enumerationAndChronology``.
+   -  ``supplementaryContent``.
+   -  ``index``.
+   -  ``missing_issues``.
+   -  ``notes``.
+
+-  Removes conditionality of the 2nd call number depending on an
+   existing 1st call number.
+-  Extends availability of the 2nd call number to all holdings types.
+-  Prevents deleted serial issue to be displayed on the public
+   interface.
+-  Harmonizes the use of JSON schema custom options for the editor for
+   enumeration.
+
+Data
+~~~~
+
+-  Adds templates records fixtures.
+-  Fixes ``import_users`` CLI to prevent storing pids in the
+   ``patron_pid`` database table, because it results in issues when
+   creating new user after the initial setup.
+-  Removes the ``append`` argument from the ``import_users`` function.
+-  Adds a username to the user fixtures.
+
+Search
+~~~~~~
+
+-  Fixes ElasticSearch bulk indexing to improve performance of parallel
+   indexing with MEF authority link creation.
+-  Adds facets to the templates search view, to filter results by
+   resource type (document, holdings, item, patron) and visibility
+   (public, private).
+-  Improves ES mapping configuration for users to prevent that searching
+   for patron by barcode retrieves multiple results, if the barcode
+   contains dashes.
+-  Updates the total results display to ElasticSearch ``7.9.0``.
+
+User management
+~~~~~~~~~~~~~~~
+
+-  Moves the user personal data from the user resource (JSON of the user
+   module) to the RERO ID profile (the user profile database).
+-  Extends the user resource with the following fields for patron:
+
+   -  Notes (displayed in the patron profile).
+   -  Expiration date (displayed in the patron profile).
+   -  Library affiliation.
+
+-  Group the patron data of the user in a nested structure.
+-  Adds a new Invenio account login view REST API.
+-  Moves the link between the patron record and the user profile from
+   the email to the id.
+-  Synchronizes the patron record and RERO ID profile data in both
+   directions.
+-  Sets the default user password as the birth date.
```

Comment:
My curiosity: birth date or birth year?
If the former, date format variations may be an issue (error-prone), but I reckon this is a PO discussion.
(FWIW, in my local library it's birth year.)

Reply:
@mmo Actually, it's birth date. Maybe @jma can tell how he handles this.